src/View/ViewableData.php

```diff
@@ -385,10 +385,11 @@ public function setCustomisedObj(ViewableData $object)
      * for a field on this object. This helper will be a subclass of DBField.
      *
      * @param string $field
-     * @return string Casting helper As a constructor pattern, and may include arguments.
+     * @param bool $useFallback If true, fall back on the default casting helper if there isn't an explicit one.
+     * @return string|null Casting helper As a constructor pattern, and may include arguments.
      * @throws Exception
      */
-    public function castingHelper($field)
+    public function castingHelper($field, bool $useFallback = true)
```

Comment:
Opted to add an extra param here instead of just never including defaults from this method because this reduces upgrade pains - any usage of this method outside of the `obj()` method should always fall back on the default casting.